README.md

```diff
@@ -53,6 +53,10 @@ The account number has been set on field 'telephone-toggle-text-2' using validat
 
 We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/your/project/tags).
 
+## UI Changes 
+
+A 'county or state' field has been added to the address page. The postcode field on the address page has been updated to 'Postcode or ZIP Code'. 
```

Comment:
Hi @al01010 what was the rationale to add this to the Readme page?

Reply:
When I went to make the pull request, I saw the following link, and it suggested updating the readme with the details of are any changes to the interface. I thought that adding the extra field would count as a change to the interface but I'm happy to remove it if you think it is unnecessary. 

https://github.com/UKHomeOffice/gro/blob/master/CONTRIBUTING.md

Reply:
interesting, I think it's good to amend the Readme in certain circumstances e.g. if you've changed the way the app runs.  Usually, I've not seen types of changes in the ReadMe though.

Note: we have all your changes in git

Reply:
I'm happy to remove it then. I haven't seen what I did in the other Readme examples I looked at and since it is a small change I don't think it needs to be there.